web/premises/views.py

```diff
@@ -216,6 +216,11 @@ def get_contentions(self, paginate=True):
                        .filter(is_featured=True)
                        .order_by("-date_modification"))
 
+        if not contentions:
```

Comment:
`not contentions.exists()`

would be better.

Reply:
I've changed it.

Reply:
Thanks 😄